mne/viz/topomap.py

```diff
@@ -3081,14 +2972,6 @@ def plot_regression_weights(
     %(ch_type_topomap)s
     %(sensors_topomap)s
     %(show_names_topomap)s
-
-        .. versionadded:: 1.2
```

Comment:
nb: this removal is not a mistake; the whole function was added in 1.2 so no need to point out that this param was also added then.